ESP8266wifi.cpp

```diff
@@ -591,7 +595,7 @@ WifiMessage ESP8266wifi::getIncomingMessage(void) {
 }
 
 // Writes commands (from PROGMEM) to serial output
-void ESP8266wifi::writeCommand(const char* text1 = NULL, const char* text2) {
```

Comment:
The signature is perfectly ok: The compiler complains about redefinitions in the implmentation.